digitalpublicgoods/openrims.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "OpenRIMS",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.openrims.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/MSH/Pharmadex2"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "The system uses REST API"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Servers located in the United States and is subject to the applicable state and federal laws of the United States."
+    ],
+    "adherenceSteps": [
+      "https://openrims.org"
```

Comment:
Condsidering PII data is collected, it would be helpful to include a privacy policy to help understand how customer or client data is managed. Thanks.